### ISSUE TITLE
"sudo runc run <container_id>" generates error 

### ISSUE THREAD

**Issue by amorphid** [opencontainers/runc#953]
## Summary

`$ sudo runc run <container_id>` generates error 

Tracked it down to commit [8dd3d634556427c22769125ef2b0aa2ce613a2a7](https://github.com/opencontainers/runc/commit/8dd3d634556427c22769125ef2b0aa2ce613a2a7)

```
$ sudo runc run fresh_debian_jessie_install
process_linux.go:258: applying cgroup configuration for process caused "invalid path for memory cgroups: &{root:/sys/fs/cgroup innerPath:/fresh_debian_jessie_install config:0xc820011950 pid:12368}"
```

boots to container just fine on previous commit [9d7831e41d3ef428b67685eeb27f2b4a22a92391](https://github.com/opencontainers/runc/commit/9d7831e41d3ef428b67685eeb27f2b4a22a92391)

```
$ sudo runc run fresh_debian_jessie_install 
# 
(^^^ container command prompt)
```
## Reproducing

**Setup**
- Running [this Debian Jessie 64 Vagrant box](https://atlas.hashicorp.com/debian/boxes/jessie64/versions/8.5.2) in Virtual Box on Mac.
- Ran [the script in this Gist](https://gist.github.com/amorphid/92f53e8e0fd7d071dc2c8125bedfb225) to install Debootstrap, Go, and Runc 

**Steps to reproduce**

```
$ cd ~
$ sudo mkdir -p containers/jessie
$ sudo debootstrap jessie containers/jessie/rootfs
$ cd containers/jessie
$ sudo runc spec
$ sudo runc run fresh_debian_jessie_install
```

**Comment by crosbymichael** [comment]
ping @vishh

**Comment by crosbymichael** [comment]
@amorphid thanks for bisecting the commits for us

**Comment by vishh** [comment]
I will look at this issue today.

**Comment by cyphar** [comment]
I think we have another issue caused by the same commit. Should we revert it until it's fixed? #946

**Comment by hqhq** [comment]
Codes caused this issue was reverted by #961 and fixed by #962 afterward.
